src/SymptomHistory/symptomHistory.ts

```diff
@@ -0,0 +1,155 @@
+import dayjs from "dayjs"
+import { Posix, isSameDay } from "../utils/dateTime"
+
+import * as Symptom from "./symptom"
+
+export interface NoData {
+  kind: "NoData"
+  date: Posix
+}
+
+export interface Symptoms {
+  id: string
+  kind: "Symptoms"
+  date: Posix
+  symptoms: Set<Symptom.Symptom>
+}
+
+export type SymptomEntry = NoData | Symptoms
```

Comment:
This may want to be:

`export type SymptomEntry = NoUserInput | UserInput`

Not sure about this but we can discuss in morning.